app/src/main/java/io/github/droidkaigi/confsched2018/util/ext/TextViewExt.kt

```diff
@@ -1,10 +1,58 @@
 package io.github.droidkaigi.confsched2018.util.ext
 
+import android.app.SearchManager
+import android.content.Intent
+import android.view.ActionMode
+import android.view.Menu
+import android.view.MenuItem
 import android.widget.TextView
+import io.github.droidkaigi.confsched2018.R
 
 // Prevent requestLayout()
 fun TextView.setTextIfChanged(newText: String) {
     if (newText != text) {
         text = newText
     }
 }
+
+val TextView.selectedText : CharSequence
+    get() = text.subSequence(selectionStart, selectionEnd)
+
+fun TextView.setTextIsSelectable(selectable: Boolean, withWebSearch: Boolean = false) {
```

Comment:
Probably we can do this with DataBindingAdapter. 
What do you think? 😄

Reply:
Indeed. I rewrite using DatabinBindingAdapter.